query-engine/connectors/sql-query-connector/src/database/transaction.rs

```diff
@@ -191,16 +192,21 @@ impl<'tx> WriteOperations for SqlConnectorTransaction<'tx> {
         .await
     }
 
-    async fn execute_raw(&mut self, query: String, parameters: Vec<PrismaValue>) -> connector::Result<usize> {
+    async fn execute_raw(&mut self, inputs: HashMap<String, PrismaValue>) -> connector::Result<usize> {
         catch(self.connection_info.clone(), async move {
-            write::execute_raw(&self.inner, query, parameters).await
+            write::execute_raw(&self.inner, inputs).await
         })
         .await
     }
 
-    async fn query_raw(&mut self, query: String, parameters: Vec<PrismaValue>) -> connector::Result<serde_json::Value> {
+    async fn query_raw(
+        &mut self,
+        _model: Option<&ModelRef>,
+        inputs: HashMap<String, PrismaValue>,
+        _query_type: Option<String>,
```

Comment:
See my comment above, I believe this could be used to differentiate the query vs execute semantics in the sql connector.